packages/medusa/src/api/routes/admin/shipping-profiles/add-product.js

```diff
@@ -0,0 +1,24 @@
+import { MedusaError, Validator } from "medusa-core-utils"
+
+export default async (req, res) => {
+  const { profile_id } = req.params
```

Comment:
Change all `profile_id` to `profileId`

Reply:
Change all variables, that does not relate to DB, to be camelCase in the following files

Reply:
I am gonna keep `profile_id` and `option_id`. I think it reads nicer when looking at the endpoints, and since we can't define the routes `/yada/:snake_case` and then use it as `const { snakeCase } = req.params` we will go ahead and use snake case whenever we need route parameters.

Reply:
But then we need to be consistent. You have used optionId and not option_id for all the ShippingOption routes.